integration_tests/src/janus.rs

```diff
@@ -169,18 +203,21 @@ impl Janus<'static> {
         // depends on this task being defined will likely time out or otherwise fail.
         // This should become more robust in the future when we implement dynamic task provisioning
         // (#44).
-        datastore(pool, RealClock::default(), &[datastore_key])
-            .unwrap()
-            .put_task(task)
-            .await
-            .unwrap();
+        let datastore = datastore(pool, RealClock::default(), &[datastore_key]).unwrap();
+        datastore.put_task(task).await.unwrap();
+
+        let batch_discovery = Arc::new(JanusClusterBatchFetch {
+            datastore,
+            _datastore_port_forward,
```

Comment:
Does this line "use" `_datastore_port_forward` enough that we can remove the leading `_` without earning an unused variable warning?

Reply:
The local variable: yes, the struct member: no. We store the `PortForward` in the struct to manage when `kubectl port-forward` is torn down as a side effect of dropping it, but we don't read from the struct member at all.